CHANGELOG.md

```diff
@@ -1,5 +1,111 @@
+
+[kwonoj]: https://github.com/kwonoj
+[pull/109]: https://github.com/ashleygwilliams/wasm-pack/pull/109
+
+- **wrap `npm login` command - [djfarly], [pull/100]**
+
+  In order to publish a package to npm, you need to be logged in. You can now use
+  `wasm-pack login` to login to the npm (or any other) registry.
+
+[pull/100]: https://github.com/ashleygwilliams/wasm-pack/pull/100
+
+- **exit early on failure - [mgattozzi], [pull/90]**
+
+  Until now, `wasm-pack` would continue to run tasks, even if a task failed. Now- if something
+  fails, we'll exit so you don't have to wait to fix the error.
+
+[pull/90]: https://github.com/ashleygwilliams/wasm-pack/pull/90
+
+### 🤕 Fixes
+
+- **force install wasm-bindgen - [ashleygwilliams], [pull/133]**
+
+  Using an out of date version of `wasm-bindgen` can run you into a bunch of trouble. This
+  very small change should fix the large number of bug reports we received from users using
+  an out of date `wasm-bindgen-cli` by force installing `wasm-bindgen-cli` to ensure the user
+  always has the latest version. We don't expect this to be a forever solution (it's a bit
+  slow!) but it should help those who are getting started have a less rough time.
+
+[pull/133]: https://github.com/ashleygwilliams/wasm-pack/pull/133
+
+- **fix CI release builds - [ashleygwilliams], [pull/135]**
+
+  This was not working! But now it is! You can always use `cargo install` to install
+  wasm-pack, but now you can find pre-built Linux and Mac binaries in the [Releases]
+  tab of our GitHub repo.
+
+[Releases]: https://github.com/ashleygwilliams/wasm-pack/releases
+[pull/135]: https://github.com/ashleygwilliams/wasm-pack/pull/135   
+
+### 🛠️ Maintenance 
+
+- **remove `quicli` dependency - [mgattozzi], [pull/131]**
+
+  While `quicli` is a great way to get started writing a CLI app in Rust- it's not meant for 
+  large, mature applications. Now that `wasm-pack` is bigger and has many active users, we've
+  removed this dependency to unblock further development on the tool.
+
+[pull/131]: https://github.com/ashleygwilliams/wasm-pack/pull/131
+
+- **update rustfmt CI test - [djfarly], [pull/128]**
+
+  Since 0.2.0 how one should call `rustfmt` changed! We've kept it up to date so we can continue
+  to maintain conventional style in the codebase.
+
+[pull/128]: https://github.com/ashleygwilliams/wasm-pack/pull/128
+
+- **custom module for errors - [mgattozzi], [pull/120]**
+
+  Thanks to the `failure` crate, we've been playing fast and loose with errors for a bit. We're
+  finally getting serious about error handling - by organizing all of our specific errors in a
+  specific module. This will make it easier to communicate these errors out and handle new error
+  cases from future features.
+
+[pull/120]: https://github.com/ashleygwilliams/wasm-pack/pull/120
+
+### 📖 Documentation
+
+Special thanks to [data-pup] who continues to be our documentation champion! In case you missed it,
```

Comment:
This link for data-pup is now broken. It should be [data-pup](https://github.com/data-pup)